DB/main.py

```diff
@@ -0,0 +1,30 @@
+import firebase_handler as fh  # Assuming your previous code was saved in firebase_handler.py
+
+
+def main():
```

Comment:
Consider adding error handling to your functions to handle cases where Firebase operations might fail, such as network issues or authentication problems.